packages/date-time/src/components/CalendarDayGrid/CalendarDayGrid.styles.ts

```diff
@@ -241,6 +241,38 @@ export const styles = (props: ICalendarDayGridStyleProps): ICalendarDayGridStyle
       ...lastTransitionRowAnimationStyle,
       animationName: lastTransitionRowAnimationStyle.animationName + ',' + transitionRowDisappearance,
     },
+    dayMarker: {
+      width: 4,
+      height: 4,
+      backgroundColor: palette.neutralSecondary,
+      borderRadius: '100%',
+      bottom: 1,
+      left: 0,
+      right: 0,
+      position: 'absolute',
+      margin: 'auto',
+      selectors: {
+        '[class*=dayIsToday] &': {
```

Comment:
anyone have suggestions of better ways to do this? this is normally where I'd just stick an "!important" on the styles, but this is an alternative way to force specificity when both "dayIsToday" and "dayMarker" classes are present

Reply:
you should be defining and using global classname: https://github.com/microsoft/fluentui/blob/88d22c29939759ac38c15333f8efa5d3f2d93fbb/packages/date-time/src/components/CalendarDayGrid/CalendarDayGrid.styles.ts#L15